<?xml version="1.0" ?>
<robot name="urc_arm">
    <group name="arm">
        <joint name="joint_1"/>
        <joint name="joint_2"/>
        <joint name="joint_3"/>
        <joint name="joint_4"/>
        <joint name="joint_5"/>
        <chain base_link="base_link" tip_link="end_effector_link"/>
    </group>

    <group_state name="home" group="arm">
        <joint name="joint_1" value="0"/>
        <joint name="joint_2" value="0"/>
        <joint name="joint_3" value="0"/>
        <joint name="joint_4" value="0"/>
        <joint name="joint_5" value="0"/>
    </group_state>
    
    <group_state name="ready_to_type" group="arm">
        <joint name="joint_1" value="0"/>
        <joint name="joint_2" value="0.5"/>
        <joint name="joint_3" value="1.0"/>
        <joint name="joint_4" value="-1.5"/>
        <joint name="joint_5" value="0"/>
    </group_state>

    <disable_collisions link1="base_link" link2="shoulder_link" reason="Adjacent"/>
    <disable_collisions link1="shoulder_link" link2="upper_arm_link" reason="Adjacent"/>
    <disable_collisions link1="upper_arm_link" link2="forearm_link" reason="Adjacent"/>
    <disable_collisions link1="forearm_link" link2="wrist_link" reason="Adjacent"/>
    <disable_collisions link1="wrist_link" link2="end_effector_link" reason="Adjacent"/>
</robot>